### DISCUSSION TITLE
How to create a project and apps in Rancher Desktop

### DISCUSSION THREAD

**Discussion by JCzz** [rancher-sandbox/rancher-desktop#4584] [General]
Hi

I am running on Rancher Desktop MacOs M1, how can I create a project with one or more namespaces?

I was looking for creating Apps, and maybe it is because I need a project first: https://ranchermanager.docs.rancher.com/v2.0-v2.4/how-to-guides/new-user-guides/helm-charts-in-rancher/launching-apps

Thanks

**Comment by jandubois** [comment]
It is not clear to me what you are trying to do. The docs you link to are about Rancher Manager, not Rancher Desktop.

**Reply by JCzz** [reply]
Thanks @jandubois 

Sorry, wrong link in previous.

From: https://ranchermanager.docs.rancher.com/v2.5/how-to-guides/advanced-user-guides/manage-clusters/projects-and-namespaces

I was looking for the following statement, desktop or not(nor aware that there is any difference):

`A project is a group of namespaces, and it is a concept introduced by Rancher. Projects allow you to manage multiple namespaces as a group and perform Kubernetes operations in them.`

**Reply by jandubois** [reply]
Ok, but that is still about Rancher Manager; this repo is about the Rancher Desktop project, which is completely different (providing container and Kubernetes functionality on a desktop). The only connection (besides the similar name) is that you can install Rancher Manager on top of Rancher Desktop, to run is locally.

You may be better off discussing your Rancher Manager questions in a Rancher Manager forum.
